tests/README.md

```diff
@@ -0,0 +1,67 @@
+# Aeneas test suite
+
+This folder contains the test suite for Aeneas. It is organized as follows:
+- `test_runner/`: ocaml script that orchestrates the test suite;
+- `src/`: rust files and crates used as test inputs;
+- `llbc/`: temporary directory (ignored by git) that stores intermediate files generated during tests;
+- `coq/`, `fstar/`, `hol4/`, `lean/`: one folder per backend. Each folder contains:
+   - Files generated by Aeneas from our test inputs;
+   - Handwritten proof files.
+
+# Running the test suite
+
+Running the test suite has two parts:
+- The generation of files from rust to each backend. This is done by running `make test` in the
+  project root. This will call the test runner on each file or crate in `tests/src`, which will call
+  `charon` and `aeneas` to generate the backend-specific outputs.
+- The verification of proofs for each backend. This is done by running `make verify` in the project
+  root. It will run each verifier (`fstar`, `lean` etc) on the files in the corresponding folder to
+  verify the correctness of proofs.
+
+CI does both of these things; in particular it checks that we correctly committed the generated
+files.
+
+Important note: at the moment we don't regenerate the `hol4` outputs. We do still check them with
+hol4. This is tracked in https://github.com/AeneasVerif/aeneas/issues/62.
+
+# Adding a new test
+
+Adding a new test is easy: just add a `foo.rs` file or a `foo_crate` folder in `tests/src`. `make
+test` will find the new file and call the test runner on it.
+
+To specify options for how your test should be run, see the next section.
+
+Ideally, any non-trivial change to Aeneas should have an accompanying test that exercises the new
+code. The goal of this test suite is to cover a large portion of what Aeneas can do, so we can work
+on it confidently.
```

Comment:
Maybe give somewhere the command to run a test for a file or a crate? (e.g., `make test-foo.rs` for a file, `make test-bar` for crate `bar`)